doc/tutorial/fcm_specifics

```diff
@@ -0,0 +1,1484 @@
+                <br/>
+                <br/>
+                A type of conflict that we may come across are "Text conflicts". 
+                These are returned via <code>fcm st</code> as
+                <pre>C    statusesC_text.txt </pre>
+                where column 1 has a "C" in it.
+                <br/>
+                To resolve the text conflict we can use the 3-way difference tool xxdiff.
+                <br/>
+                For each file with a text conflict, the <code>fcm conflicts</code> command 
+                calls a graphical merge tool (i.e. xxdiff by default) to display a 3-way diff.
+                <br/>
+                <br/>
+                <h4>How to use xxdiff to help resolve text conflicts</h4>
+                The file on the left is our original file.
+                The file in the middle is the common ancestor from the merge.
+                The file on the right is the file containing the changes which we are merging 
+                in.
+                <br/>
+                <br/>
+                A quick way of resolving the conflict (from Essentials tutorial III <a href="fcm_essentials#S15">text conflicts</a>):
+                <div class="panel-group" id="accordion">
+                    <div class="panel panel-default">
+                        <div class="panel-heading">
+                            <h4 class="panel-title">
+                                <a data-toggle="collapse" data-parent="#accordion" href="#collapseNine">
+                                <span class="glyphicon"></span>
+                                Answers
+                                </a>
+                            </h4>
+                        </div>
+                        <div id="collapseNine" class="panel-collapse collapse">
+                            <div class="panel-body">
+                                <div class="row">
+                                    <font color="green">
+                                    <img src="text_conflict2.png" alt="fcm cf screengrab">
+                                    <br/>
+                                    # Left click on the text option that we want to keep, it should change colour
+                                    from yellow to purple.
+                                    <br/>
+                                    # Then press "m", or use the toolbar; File -> Exit with MERGED M.
+                                    </font>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <h4>Explanations of the three window split xxdiff:</h4>
+                The number on the far right panel of xxdiff tool is the number of unselected 
+                differences, known as hunks or sections and they are often seen in yellow.
+                The correct term for the highlighted section is hunk, but here we will refer to
```

Comment:
As its the correct term and you actually say that you should probably use that rather than section.